package.json

```diff
@@ -26,9 +25,15 @@
     "opn": "^5.2.0",
     "prop-types": "^15.6.1",
     "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "react-dom": "^16.2.0",
+    "socket.io": "^2.0.4"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^3.3.14",
```

Comment:
This diffs are not on master ? 🤔

Reply:
I added dependency for socket.io which moved some dependency to the top of the file.